src/lib/dlt_user.c

```diff
@@ -4708,6 +4698,11 @@ int dlt_start_threads()
         dlt_log(LOG_CRIT, "Can't create housekeeper thread!\n");
         return -1;
     }
+    if (pthread_setname_np(dlt_housekeeperthread_handle, "dlt_housekeeper"))
```

Comment:
Any reason you moved from `dlt_user_housekeeperthread_function`?

Reply:
I do not see how to handle an error in the thread functions themselves. Also I think it's a bit nicer to do the renaming together with the thread creation.